ideascube/mediacenter/forms.py

```diff
@@ -27,3 +30,30 @@ def save(self, commit=True):
         document.save()
         self.save_m2m()
         return document
+
+
+def CreateDocumentForm(path):
+    class NotFileDocumentForm(forms.ModelForm):
+
+        class Meta:
+            model = Document
+            exclude = ['original', 'preview']
+
+        original = forms.FilePathField(path=path)
```

Comment:
I don't get it here, the path is use a the creation of the class and it come from outside.
We could use a callable but we still need to provide this custom callable at class definition.